distributed/dashboard/tests/test_worker_bokeh.py

```diff
@@ -28,37 +28,37 @@
     worker_kwargs={"dashboard": True},
     scheduler_kwargs={"dashboard": True},
 )
-def test_routes(c, s, a, b):
+async def test_routes(c, s, a, b):
     port = a.http_server.port
 
     future = c.submit(sleep, 1)
-    yield gen.sleep(0.1)
+    await gen.sleep(0.1)
```

Comment:
can gen.sleep also be replaced with asyncio.sleep ?

Reply:
sure, why not

Reply:
done!